src/index.html

```diff
@@ -5,8 +5,317 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header page__header">
+      <div class="logo header__logo">
+        <a href="#" class="logo__link">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a 
+              href="#" 
+              data-qa="nav-hover" 
+              class="nav__link"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <section class="section main__section">
+        <div class="card section__card" data-qa="card">
```

Comment:
Use `page__section` instead of `main__section`. It helps to avoid a lot of unnecessary classes and scss files for this blocks.

```suggestion
    <main>
      <section class="section page__section">
        <div class="card section__card" data-qa="card">
```

Reply:
I tried but lint doesn't pass my code